src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/BackupConfigurationIT.kt

```diff
@@ -0,0 +1,86 @@
+package com.atlassian.performance.tools.jiraactions.api.scenario
+
+import com.atlassian.performance.tools.dockerinfrastructure.api.browser.DockerisedChrome
+import com.atlassian.performance.tools.dockerinfrastructure.api.jira.JiraCoreFormula
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.measure.output.CollectionActionMetricOutput
+import com.atlassian.performance.tools.jiraactions.api.memories.User
+import com.atlassian.performance.tools.jiraactions.api.memories.UserMemory
+import com.atlassian.performance.tools.jiraactions.api.page.isElementPresent
+import com.atlassian.performance.tools.jiraactions.api.w3c.DisabledW3cPerformanceTimeline
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.openqa.selenium.By
+import java.time.Clock
+import java.util.*
+
+class BackupConfigurationIT {
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    /**
+     * During the test, you can connect to WebDriver by the VNC viewer.
+     * DockerisedChrome opens the port 5900 so executing command
+     * docker ps
+     * will show you which port on your host is open and mapped to 5900 on the container.
+     */
+    @Test
+    fun shouldDisableBackups() {
```

Comment:
The CI fails with "Gradle exited with 137", which means the OS is running out of memory.

I'll run `./gradlew build` locally and see how much memory is consumed and how long it takes.
The memory can be used by Gradle and Docker.
If it's longer than 21 minutes, we'll need to bump the `travis_wait` in `.travis.yml`.

Reply:
Ok, the local run took: `BUILD SUCCESSFUL in 19m 35s`
Before the test, Docker used 1 GB. During the test it used 6 GB. The containers were nicely cleaned up after every test, but Docker did not release the memory. It's been 5 minutes after the tests have finished and it still holds 6 GB.

Reply:
PS. I observed the build via VNC, we're accidentally testing it multiple times. `JiraCoreScenarioIT` runs the `scenario.getSetupAction` already. Perhaps integration tests are truly about integration - joining things together. In this case we could hijack the test and cover the backup service deletion, even if it's not default anymore.

Reply:
In the VNC, I also saw this:
![image](https://user-images.githubusercontent.com/1194704/62784266-699d9b80-babe-11e9-88e9-70456946f87e.png)
![image](https://user-images.githubusercontent.com/1194704/62784289-75895d80-babe-11e9-8fa0-351e71095044.png)
As if a previous `sendKeys(license)` kept a buffers of keys to send and we navigated away and kept sending it on the next screen.

Reply:
WRT memory, it seems we can tell [Java to work better with virtualization](https://stackoverflow.com/a/51364393). Note that this was about running Java in Docker, and we're running Docker via Java. *But* the CI environment is also virtualized, so it might help.
Additionally, we can set the max heap for the JUnit JVM. It seems our Travis has 7.5 GB of RAM. If 5-6 GB go for Docker, then we're left with around 1 GB for the JVM. So let's [set the max heap](https://github.com/atlassian/jira-hardware-exploration/blob/6efaff838cf59fad1e4c77276e90051827e862d0/build.gradle.kts#L41) to 1 GB or less.

Reply:
I’m using `docker stats` and according to that tool my containers are not using more memory than 2.5 GB (first ~2GB, second ~400 MB) per one test.
Docker version 19.03.1